include/bluetooth/services/fast_pair/use_case/locator_tag/fp_adv.h

```diff
@@ -0,0 +1,183 @@
+ *  the BT_ID_DEFAULT identity is used.
+ *
+ *  @param id Bluetooth identity for the Fast Pair advertising.
+ *
+ *  @return 0 if the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+int bt_fast_pair_locator_tag_fp_adv_id_set(uint8_t id);
+
+/** @brief Get the Bluetooth identity used for the Fast Pair advertising.
+ *
+ *  @return Bluetooth identity for the Fast Pair advertising.
+ */
+uint8_t bt_fast_pair_locator_tag_fp_adv_id_get(void);
+
+/** Fast Pair advertising information callback descriptor. */
+struct bt_fast_pair_locator_tag_fp_adv_info_cb {
+	/** Notify that the advertising state has changed.
+	 *
+	 * This information callback is used to track the state of the Fast Pair advertising set.
+	 * The current state can be used to signal the application mode in the user interface
+	 * (for example, on LEDs).
+	 *
+	 *  @param enable True:  Fast Pair advertising is enabled.
+	 *                False: Fast Pair advertising is disabled.
+	 */
+	void (*state_changed)(bool enabled);
+};
+
+/** @brief Register Fast Pair advertising information callbacks.
+ *
+ *  This function registers an instance of information callbacks. The registered instance needs
+ *  to persist in the memory after this function exits, as it is used directly without the copy
+ *  operation. It is possible to register only one instance of callbacks with this API.
+ *
+ *  This function can only be called before enabling Fast Pair with
+ *  the @ref bt_fast_pair_locator_tag_fp_adv_enable API.
+ *
+ *  This function must be called in the cooperative thread context.
+ *
+ *  @param cb Callback struct.
+ *
+ *  @return Zero on success or negative error code otherwise
+ */
+int bt_fast_pair_locator_tag_fp_adv_info_cb_register(
+	const struct bt_fast_pair_locator_tag_fp_adv_info_cb *cb);
+
+/** @brief Enable the Fast Pair advertising module.
+ *
+ *  Can be called after the Fast Pair advertising module has been initialized using the
+ *  @ref bt_fast_pair_locator_tag_fp_adv_init function.
+ *
+ *  Must be called after enabling the Fast Pair subsystem using the @ref bt_fast_pair_enable
+ *  function.
+ *
+ *  @return 0 if the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+int bt_fast_pair_locator_tag_fp_adv_enable(void);
+
+/** @brief Disable the Fast Pair advertising module.
+ *
+ *  Can be called after the Fast Pair advertising module has been initialized using the
+ *  @ref bt_fast_pair_locator_tag_fp_adv_init function.
+ *
+ *  Should be called before disabling the Fast Pair subsystem using the @ref bt_fast_pair_disable
+ *  function.
+ *
+ *  @return 0 if the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+int bt_fast_pair_locator_tag_fp_adv_disable(void);
+
+/** @brief Check if the Fast Pair advertising module is ready.
+ *
+ *  @return true when the module is ready, false otherwise.
+ */
+bool bt_fast_pair_locator_tag_fp_adv_is_ready(void);
```

Comment:
I also wonder if we could make the module automatically react on the factory reset (it could even clear NVM data related to the used Bluetooth identity)

Reply:
Discussed. We will keep it as convenience API.